### PR TITLE
Refresh social wall on /20years

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Refresh social wall on /20years

## QA

- Go to /20years
- Compare page against https://docs.google.com/document/d/1vlENASSxNJdnK5tRuNYEZGVbh_qrVN0lhon8SOYo6A0/edit?tab=t.0

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-15739?focusedCommentId=599065&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-599065

### CODE REVIEW

File: templates/20years/index.html [hunk 1]
```diff
@@ -897,6 +1041,43 @@ <h2>Relive your best Ubuntu memories</h2>
...
               </p>
+              <div class="p-section--shallow">
+                <hr class="p-rule--muted" />
+                <p>
+                  <strong>
+                    <a href="https://discourse.ubuntu.com/t/oracular-oriole-24-10-wallpaper-competition/46455">Oracular Oriole (24.10)</a>
+                  </strong>
+                </p>
+                <div class="row">
+                  <div class="col-3 col-medium-2">
```
Comment (minkyngkm) [inline templates/20years/index.html:1052]: I wasn't sure  about the images here. Could anyone who knows provide suggestions on which images to use?

**Comment by petesfrench** [comment]
Can you update [this line](https://github.com/canonical/ubuntu.com/blob/main/static/sass/_pattern_tube-line.scss#L33) to be 110%, it should fix a visual bug that is occuring

**Comment by petesfrench** [comment]
These images aren't loading for me
image

File: templates/20years/index.html [hunk 2]
```diff
@@ -773,32 +765,81 @@ <h2>
             </div>
           </div>
         </div>
+        <div class="p-card--social-media">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img"
```
Comment (petesfrench) [inline templates/20years/index.html:771]: This image isn't loading for me

File: templates/20years/index.html [hunk 3]
```diff
@@ -807,27 +848,28 @@ <h2>
           <div class="p-heading-icon">
             <div class="p-heading-icon__header">
               <img class="p-heading-icon__img"
-                   src="https://assets.ubuntu.com/v1/3aa35ae4-@nebojsa%20tomic.png"
+                   src="https://assets.ubuntu.com/v1/5e03e442-user.svg"
```
Comment (petesfrench) [inline templates/20years/index.html:851]: This image isn't loading for me

File: templates/20years/index.html [hunk 4]
```diff
@@ -708,31 +708,23 @@ <h2>
...
-            {{ image(url="https://assets.ubuntu.com/v1/081ef9b7-post%203.png",
-                        alt="",
-                        width="1512",
-                        height="851",
-                        hi_def=True,
-                        loading="lazy") | safe
-            }}
-          </div>
+          <p>20 years of Ubuntu , the OS that we all love ❤️</p>
+          <p>What's your favourite release?</p>
```
Comment (petesfrench) [inline templates/20years/index.html:719]: This isn't in the original post https://www.linkedin.com/posts/shannonandriansmith_ubuntults-opensource-activity-7191994640412151808-KUQW?utm_source=share&utm_medium=member_desktop

File: templates/20years/index.html [hunk 5]
```diff
@@ -773,32 +837,86 @@ <h2>
...
-                <strong><small>Tweet by
-                @missingfaktor</small></strong>
+                <strong><small>LinkedIn post by Mohsen Mostafa Jokar</small></strong>
               </p>
             </div>
           </div>
-          <p>Happy 20 years, Ubuntu! 🎉</p>
           <p>
-            I still fondly look back on my first installation with 7.04 Feisty Fawn. It's amazing to see how far Ubuntu has come since then. Here's to many more years! 🧡🤎💜
+            20 years of Ubuntu: Canonical celebrates with upcoming 24.10 - and it's outstanding as usual <br/> #Linux hashtag #Ubuntu hashtag #Desktop hashtag #Canonical
```
Comment (petesfrench) [inline templates/20years/index.html:899]:
```suggestion
            20 years of Ubuntu: Canonical celebrates with upcoming 24.10 - and it's outstanding as usual <br/> &#35;Linux &#35;Ubuntu &#35;Desktop &#35;Canonical
```
Could use encodings here (and everywhere else)

**Comment by petesfrench** [comment]
In this post there is a space missing ' provides. Congratulations'. (my mistake)
image

File: templates/20years/index.html [hunk 6]
```diff
@@ -667,62 +667,99 @@ <h2>
...
+            Releases that stick out for me: 6.06 | 12.04 | 20.04
           </p>
           <div class="p-section--shallow">
-            {{ image(url="https://assets.ubuntu.com/v1/a5a694bc-post 8.png",
-                        alt="",
-                        width="1512",
-                        height="852",
-                        hi_def=True,
-                        loading="lazy") | safe
+            {{ image (
```
Comment (petesfrench) [inline templates/20years/index.html:684]:
```suggestion
            {{ image(
```
This is going to sound super annoying, but you have to remove the space and the run djlint. I know it seems silly (I pushed back on it), but it is what the team agreed on

**Review by petesfrench — APPROVED** [review]
LGTM, just a couple minor things

**Comment by petesfrench** [comment]
I wonder if there is a way to order these so that this one card isn't hanging out on its own

**Comment by minkyngkm** [comment]
@petesfrench I've rearranged the social wall. Would it look better?

**Review by petesfrench — APPROVED** [review]

**Pull request merged**